.pre-commit-config.yaml

```diff
@@ -17,9 +17,7 @@ repos:
     hooks:
       - id: codespell
         args:
-          - --ignore-words-list=hass,alot,datas,dof,dur,ether,farenheit,hist,iff,iif,ines,ist,lightsensor,mut,nd,pres,referer,rime,ser,serie,te,technik,ue,uint,visability,wan,wanna,withing,iam,incomfort,ba,haa,pullrequests
-          - --skip="./.*,*.csv,*.json"
-          - --quiet-level=2
+          - --config=setup.cfg
```

Comment:
Args should not be needed with this configuration anymore.

Reply:
Refer to answer: https://github.com/home-assistant/core/pull/73940#discussion_r906002128